### ISSUE TITLE
Logo icons - broken CDN path

### ISSUE THREAD

**Issue by dev-gian** [atisawd/boxicons#22]
Hey, 

logo icons seem to not be working properly in v1.9.
I'm currently on a Vue project and importing the Web Component as such
`import 'boxicons'`
However, when attempting to display any of the logo icons (eg: `<box-icon type="logo" name="twitter"></box-icon>`), I get this in console: 
`Error: 404 Cannot find "/svg/logos/bxl-facebook-square.svg" in boxicons@1.9.0`

Changing **/logos/** to **/logo/** fixes the issue.

**Comment by atisawd** [comment]
I will look into this now thanks

**Comment by purtuga** [comment]
Hmmm... 
FYI: renaming something is ok as a Patch as long as you don't remove the prior named icon...  Please consider these backward incompatible/backbreaking changes in the future and instead issue a Major release instead of a patch/minor version. The community will THANK YOU for it :smile: and will not be surprise when out of no where, their applications break.

**Comment by atisawd** [comment]
yup i had to change the border radius of icons for it to look better. Had to redo a lot of them , so only included a few essential icons . I should have announced it to the community  before hand

**Comment by purtuga** [comment]
No problem @atisawd - I just did not want folks to miss-trust the version numbering for your releases as you continue to make improvements...  In these cases its important that a major version be used because most developers will allow auto-installs of minor and patches (using the `^` symver indicator) and thus will start to experience breakage.

Thanks again for all your efforts on this library and for sharing it with the open source community. It has been very useful and is my "goto" icon library of choice.

**Comment by dev-gian** [comment]
Agree with @purtuga, and thanks for the update!
